salt/modules/dockerio.py

```diff
@@ -648,8 +675,9 @@ def create_container(image,
         }
         __salt__['mine.send']('docker.get_containers', host=True)
         return callback(status, id_=container, comment=comment, out=out)
-    except Exception:
+    except Exception, e:
```

Comment:
Is it possible for you to target a more specific exception here?  Also this needs to be in the form `except Exception as exception:` so that it is python 3 compatible, which we are gradually making progress towards.